support/certs/tls.go

```diff
@@ -45,6 +47,13 @@ const (
 	TLSSignerKeyMapKey = "tls.key"
 	// UserCABundleMapKeyis the key value in a user-provided CA configMap.
 	UserCABundleMapKey = "ca-bundle.crt"
+	// Custom certificate validity. The format of the annotation is a go duration string with a numeric component and unit.
+	// Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h"
+	CertificateValidityAnnotation = "hypershift.openshift.io/certificate-validity"
+	CertificateValidityEnvVar     = "CERTIFICATE_VALIDITY"
+	// Custom certificate renewal percentage. The format of the annotation is a float64 value between 0 and 1.
```

Comment:
nit: "certificate renewal percentage" doesn't really explain well what the percentage is for. 
It might be easier to explain by an example. The certificate will renew when there is less than [this percentage] of the validity period left in a certificate's validity. For example... you set the validity to 100 days, and the percentage validity at "0.30".  It means that when there are less than  [ (100 days * .30) = 30 ] 30 days before the certificate expires, the certificate will renew.